### ISSUE TITLE
iOS TextInput Prevents Swipe Gesture Being Passed To List

### ISSUE THREAD

**Issue by sundayz** [jemise111/react-native-swipe-list-view#372]
I’m not 100% certain if this is a React Native issue or an issue with this package. On iOS if you have a TextInput inside a row of the swipe list, you cannot swipe left/right over the text input. You have to start the gesture outside of the text input. This isn’t an issue on android. You can begin the gesture on the text input and swipe just fine. 

Tapping the text box is fine so it looks like the TextInput is “eating” the swipe event and prevents it from being passed to the component under it. I tried finding a prop or setting which lets TextInput ignore swipe gestures while still accepting taps but I cannot seem to find one. 

Any ideas?

**Comment by jemise111** [comment]
Hey @sundayz I didn't know about this prop until now but check this out:

https://facebook.github.io/react-native/docs/textinput#rejectrespondertermination

Sounds like exactly what you need

**Comment by sundayz** [comment]
That's 100% what I need! Thanks so much, I have no idea how I looked over it, I was scrolling down that page just a few hours ago looking for a prop like that. At least others having this problem will find this issue on google.

One slight problem, the documentation seems to have it the other way around. When I set `rejectResponderTermination={false}` it allows textinput to pass the swipe events, and when it's true it seems to prevent swipes from going through.
